README.md

```diff
@@ -67,12 +80,17 @@ Before running the tests, a one-time set up is needed to fetch development depen
 
 The browser extensions are distributed through the stores of each vendor (see links above). But the packages are also saved in https://download.kiwix.org/release/browsers/ if necessary.
 
-Some nightly builds are generated, and should only be used for testing purpose: https://download.kiwix.org/nightly/
+Some nightly builds are generated, and should only be used for testing purpose: https://download.kiwix.org/nightly/.
+
+There is a test implementation of the latest code at https://kiwix.github.io/kiwix-js/, but this is used for development, and may be buggy, experimental
+or unstable.
 
 ## Previous versions
 
-The first versions of this application were originally part of the Evopedia project: http://www.evopedia.info (now discontinued). There was a "articles nearby" feature, that was able to find articles around your location. It has been deleted from the source code with everything related to Evopedia (but still in git history in versions<=2.0.0)
+The first versions of this application were originally part of the Evopedia project: http://www.evopedia.info (now discontinued). There was a "articles nearby" feature, that was able to find articles around your location. It has been deleted from the source code with everything related to Evopedia (but still in git history in versions<=2.0.0).
+
 These first versions were targeting Firefox OS (now discontinued too: we're not lucky ;-) ).
```

Comment:
We can remove the "now"

Reply:
Ah I removed a "now" above, but forgot this one.